crates/ruma-events/src/room/message.rs

```diff
@@ -721,6 +721,7 @@ impl MessageFormat {
 /// Common message event content fields for message types that have separate plain-text and
 /// formatted representations.
 #[derive(Clone, Debug, Deserialize, Serialize)]
+#[cfg_attr(not(feature = "unstable-exhaustive-types"), non_exhaustive)]
 pub struct FormattedBody {
```

Comment:
This type only serves to more strongly represent the connection of `format` & `formatted_body`. I don't think it will ever contain more than these two fields, so it doesn't need to be non-exhaustive.